packages/runtime-core/src/hydrationStrategies.ts

```diff
@@ -1,6 +1,28 @@
 import { isString } from '@vue/shared'
 import { DOMNodeTypes, isComment } from './hydration'
 
+// https://github.com/nuxt/nuxt/blob/main/packages/nuxt/src/app/compat/idle-callback.ts
+// Polyfills for Safari support
+// https://caniuse.com/requestidlecallback
+const requestIdleCallback: Window['requestIdleCallback'] =
+  globalThis.requestIdleCallback ||
```

Comment:
Can't assume `globalThis` presence given Vue's support range. Should use `import { getGlobalThis } from '@vue/shared'`

Reply:
Thanks for reviewing, I used the import